src/NonPhpFile/NeonMethodCallRenamer.php

```diff
@@ -0,0 +1,52 @@
+<?php
+declare(strict_types=1);
+
+namespace Rector\Core\NonPhpFile;
+
+use Rector\Core\Contract\Processor\FileProcessorInterface;
+use Rector\Core\ValueObject\NonPhpFile\NonPhpFileChange;
+use Rector\Renaming\Configuration\MethodCallRenameCollector;
+use Symplify\SmartFileSystem\SmartFileInfo;
+
+class NeonMethodCallRenamer implements FileProcessorInterface
+{
+    /**
+     * @var MethodCallRenameCollector
+     */
+    private $methodCallRenameCollector;
+
+    public function __construct(MethodCallRenameCollector $methodCallRenameCollector)
+    {
+        $this->methodCallRenameCollector = $methodCallRenameCollector;
+    }
+
+    public function process(SmartFileInfo $smartFileInfo): ?NonPhpFileChange
+    {
+        $oldContent = $newContent = $smartFileInfo->getContents();
+
+        foreach ($this->methodCallRenameCollector->getMethodCallRenames() as $methodCallRename) {
+            $oldObjectType = $methodCallRename->getOldObjectType();
+
+            $className = str_replace('\\', '\\\\', $oldObjectType->getClassName());
```

Comment:
Is it possible to find all children of given class / interface? In rename method rector there could be some interface, but in config.neon we will have its implementation, so the following pattern find nothing

Reply:
you may can use NodeRepository, see https://github.com/rectorphp/rector/blob/1e4a0745a140c59346f5a119f974a76bc083c58e/packages/NodeCollector/NodeCollector/NodeRepository.php#L365

Reply:
Unfortunately it is not working (at least in tests), I also tried get_declared_classes() or ExistingClassesProvider but those load only some classes not those which are in Source directory of tests

Reply:
Yes, it seems that can't collect class/interface in `Source` directory right now, you probably need to use in one fixture, eg:

```php
class AnotherClass
{}

class Fixture
{
   public function run()
  {
         new AnotherClass();
  }
}

```

Reply:
that can be problem :) My fixture is neon file (config similar to yaml). I don't have php here

Reply:
You'd have to add such feature then

Reply:
this is not neon-related, I know class / interface name (as string) and want to get list of its children (including it self). I cannot believe you never need it :)

btw I tried NodeRepository::findClassesAndInterfacesByType(AbstractRector::class); and it found nothing too. So I don't know if I used it wrong or it didn't work at all

Reply:
This requires to load classes with robot loader or using `composer dump-autoload -ao`. That's why we dropped removing unused private classes and similar :)

Personally I would skip it in MVP to make merge easier.